Code/Engine/RendererCore/Shader/ShaderStageBinary.h

```diff
@@ -38,6 +38,7 @@ class EZ_RENDERERCORE_DLL ezShaderConstantBufferLayout : public ezRefCounted
         Mat4x4,
         Transform,
         Bool,
+        Struct,
```

Comment:
We don't support structs in constant buffers

Reply:
Will revise this later.